openstudioreleases/README.md

```diff
@@ -106,8 +172,6 @@ yarn es scripts/getReleaseInfo.ts
 yarn es scripts/syncReleaseAssets.ts
 ```
 
-## Google Analytics
-
-Set the `GA_ID` in the `.env` file
+# TODO: explain this
```

Comment:
I also don't get the thing about 

> Upload `cors.json` to the bucket.

Where is that cors.json?!

Reply:
It's at https://github.com/openstudiocoalition/openstudiocoalition.github.io/blob/master/openstudioreleases/config/cors.json